src/components/search_list.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const SearchList = (props) => {
+  const handleClick = (guestHouse) => {
+    props.updateCurrent(guestHouse);
+  };
+  console.log(props);
+
+  return (
+    <div>
+      {props.db.map((guestHouses) => {
+        const boundItemClick = handleClick.bind(this, guestHouses.name);
+        return (
+          <button
```

Comment:
maybe this button should be a different component and then the bind function wont be inside the map function